fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -180,9 +180,132 @@ def additional_upload_parameters
     end
   end
 
+  # Generates commands and executes the altool.
+  class AltoolTransporterExecutor < TransporterExecutor
+    ERROR_REGEX = /\*\*\* Error:\s+(.+)/
+
+    private_constant :ERROR_REGEX
+
+    def execute(command, hide_output)
+      if Helper.test?
+        yield(nil) if block_given?
+        return command
```

Comment:
```suggestion
        command
```

Reply:
This cannot be done otherwise it will be problem on the test case. Actually, this function will return `true` or `false` for normal case but this should return command string if test and implicit return cannot be used at the case.